docs/contracts/functions.rst

```diff
@@ -319,6 +319,10 @@ will consume more gas than the 2300 gas stipend:
     not recommended, since the fallback is invoked and would not fail for interface confusions
     on the part of the sender).
 
+.. warning::
+    Note that ``send`` and ``transfer`` are deprecated and scheduled for removal in the next breaking version (0.9).
```

Comment:
I still think this should be the first of the three warnings as it's probably the most relevant one. The text also heavily focuses on using `send` and `transfer` - which ideally should be changed to call with a link to the appropriate section in the documentation. This could be part of the follow-up? :)

Reply:
True. Moved it to be the first warning.
> This could be part of the follow-up? :)

I think these should be in the follow-up because we can then already have the compiler emitting the warnings while we adapt the docs for the breaking release.